games-action/flashback/flashback-0.5.2.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="Based on REminiscence REinforced"
+DESCRIPTION="REminiscence is a re-implementation of the engine used in
+the game 'Flashback: The Quest For Identity' made by Delphine Software and released in 1992.
+REinforced is an enhanced version of the original REminiscence.
+
+You will need the original files of the PC (DOS or CD), Amiga or Macintosh release."
+HOMEPAGE="http://cyxdown.free.fr/reminiscence"
+COPYRIGHT="2005-2021 Gregory Montoir
+	2020-2021 Alex Chermenin"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/chermenin/REminiscence/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="c3b5a0d074e6846499e641fee27384b0a09769be6634215abd82d4d78691f8e8"
+SOURCE_DIR="REminiscence-$portVersion"
+ADDITIONAL_FILES="flashback.hvif
+	flashback.sh"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	flashback$secondaryArchSuffix = $portVersion
+	app:Flashback = $portVersion
+	cmd:flashback
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	tremor$secondaryArchSuffix
+	lib:libmodplug$secondaryArchSuffix
+        lib:libogg$secondaryArchSuffix
+        lib:libSDL2_2.0$secondaryArchSuffix
+        lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	tremor${secondaryArchSuffix}_devel
+	devel:libmodplug$secondaryArchSuffix
+	devel:libSDL2_2.0$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	make $jobArgs
+}
+
+INSTALL()
+{
+	mkdir -p $binDir $appsDir
+	cp fb "$appsDir/Flashback"
+	cp $portDir/additional-files/flashback.sh "$binDir/flashback"
```

Comment:
Better copy this to `$prefix/bin`, doing this in `$bindir` ends up in `$prefix/bin/x86` on 32bit secondary architecture and then needs `cmd:flashback$secondaryArchSuffix` in PROVIDES then.